### PR TITLE
Fix firedrake-zenodo path finding

### PR DESCRIPTION
**Pull request by dham**

### CODE REVIEW

**Comment by miklos1** [comment]
OK.

File: scripts/firedrake-zenodo [hunk 1]
```diff
@@ -97,6 +93,13 @@ else:
                         format='%(message)s')
 log = logging.getLogger()
 
+cwd = os.getcwd()
+try:
+    src = os.environ["VIRTUAL_ENV"] + "/src"
```
Comment (miklos1) [inline scripts/firedrake-zenodo]: What if the user has chosen global installation? Is that still supported?

Comment (dham) [inline scripts/firedrake-zenodo]: I don't know how to find the source code for a global installation, which makes firedrake-zenodo rather hard to do.

Comment (wence-) [inline scripts/firedrake-zenodo]: I think that the install procedure would have to encode where it was installed *from*.

**Pull request merged**